source/how-govuk-sign-in-works.html.md.erb

```diff
@@ -1,68 +1,46 @@
 review_in: 6 months
 ---
 
 # How GOV.UK Sign In works
-GOV.UK Sign In is an [OpenID Connect (OIDC)](https://openid.net/connect/)-compliant service that helps you authenticate your users who are using their GOV.UK Account.
-
-GOV.UK Sign In’s core functionality works in 3 steps.
-
-1. Sending your users to GOV.UK Sign In to begin authentication.
-1. Getting your user’s consent to share their data.
-1. Generating tokens to get information about your users.
-
-You can read [guidance about cookies on GOV.UK](https://www.gov.uk/help/cookie-details) if you want to learn more about cookies for GOV.UK Accounts.
-
-## Sending your users to GOV.UK Sign In to begin authentication
-
-GOV.UK Sign In uses OpenID Connect Authorization Code Flow to authenticate your users.
-
-<%= image_tag "sending-your-users-to-begin-authentication.svg", { :alt => '' } %>
-
-1. Your user selects the button on your service, which causes your service to call the `/authorize` endpoint to start the sign in process.
-1. Your user is redirected to GOV.UK Sign In to begin the authentication process.
-1. If your user is not already logged in, GOV.UK Sign In asks them to authenticate themselves.
-1. Your user either creates a GOV.UK Account, or enters their credentials and submits them to GOV.UK Sign In.
-1. GOV.UK Sign In validates the user’s credentials.
-
-## Getting your user’s consent to share their data
-
-If your user is using your service for the first time, GOV.UK Sign In will send a request to your user to ask if they agree to share their data.
-
-<%= image_tag "getting-user-permission.svg", { :alt => '' } %>
 
+GOV.UK Sign In is an [OpenID Connect (OIDC)](https://openid.net/connect/)-compliant service that helps you authenticate your users who are using their GOV.UK Account.
 
-1. GOV.UK Sign In shows your user a list of attributes your service has requested.
-1. Your user can choose to accept or reject sharing the attributes with your service.
-1. GOV.UK Sign In records the user’s choice to accept or reject.
+GOV.UK Sign In uses 2 different environments:
 
-You can read more about [choosing which user attributes your service can request](/integrate-with-integration-environment/choose-which-user-attributes-your-service-can-request/).
+* an integration environment, which contains sample users and data for you to test your service’s integration with GOV.UK Sign In
+* a production environment, which is the live environment for real users to access and use your service’s integration with GOV.UK Sign In
 
-## Generating tokens to get information about your users
+## Understand the flow GOV.UK Sign In uses
 
-GOV.UK Sign In will generate:
+<%= image_tag "top-level-technical-diagram.svg", { :alt => '' } %>
 
-* an ID token, which contains the unique subject identifier (UUID) for your user
-* an access token, which your service can use to access additional attributes (if the user has given permission)
+1. Your service asks the user to sign in or create an account.
```

Comment:
I assume that because this is more a summary of a flow process instead of a detailed step through a process that a user has to complete, we're not using bold text for UI elements as there are none here, strictly?

Reply:
Maybe we could say "sign in to or create an account", just so the words either side of "or" work

Reply:
Cheers Jon - yes, this wouldn't work for bold UI text as it's a flow process. I don't think 'sign in to' quite works though ... is 'sign in' not clear enough? Happy to chat if needed to unpick this further

Reply:
it's more than fine, just a suggestion

Reply:
thank you!